### ISSUE TITLE
[Feature] Folder enhancements

### ISSUE THREAD

**Issue by Estebiu** [hotsno/mpv-anilist#1]
There are sometimes when multiple episodes can be scattered accross different folders; like with the K-ON!! release by Tsundere.  

`Unmapped folders:
[1] K-ON/[Tsundere] K-On!! - 13-15 [BDRip h264 1920x1080 10bit FLAC]
[2] K-ON/[Tsundere] K-On!! - 10-12 [BDRip h264 1920x1080 10bit FLAC]
[3] K-ON/[Tsundere] K-On!! - 07-09 [BDRip h264 1920x1080 10bit FLAC]
[4] K-ON/[Tsundere] K-On!! - 04-06 [BDRip h264 1920x1080 10bit FLAC]
[5] K-ON/[Tsundere] K-On!! - 01-03 [BDRip h264 1920x1080 10bit FLAC]
[6] K-ON/[Tsundere] K-On! [BDRip h264 1920x1080 FLAC]/SP`

I don't know if the adoption of anitomy will somehow fix things by grouping them togheter, but a similar project, TRACKMA, didn't recognize them as one show.

Another improvement would be adding more than 1 anime folder to the config.json, as many have quite a lot of anime stored on their systems.

For the last thing, I don't know why but everytime I use the script it returns me "You have nothing to watch! (or AniList status isn't "WATCHING")"

**Comment by Estebiu** [comment]
And btw your website is fantastic, do you mind open-sourcing it?

**Comment by hotsno** [comment]
Hey! Thanks for checking the project out ☺️

I now realize that I totally should have better described what "Anime folder path" means when the setup is being run... This folder is meant to house _all_ your anime, not just one, if that's what your impression was (though looking at it now, it seems like your anime folder is the parent directory of `K-ON`, so you might have figured that out already).

Here is the folder I have added as an example:
<img width="1032" alt="image" src="https://user-images.githubusercontent.com/71658949/179287024-10a1dc81-4d87-4799-98b5-4e6a85c7aacc.png">

Note that `mpv-anilist` will only treat the "deepest" folders within each subfolder of the "anime folder" as entries. E.g. If you have a directory called `Kaguya-sama - Love is War (2019)` inside of the anime folder, with 3 more folders `Season 1`, `Season 2`, `Season 3` inside it, it will treat each of those as a separate entry.

Unfortunately, that means that in the case of this release where episodes are scattered across multiple folders, this doesn't work, but honestly that's my first time every seeing a release that does that. For the time being, if you want to get this to work with this specific case, you'll need to put all of the episodes under one directory.

> For the last thing, I don't know why but everytime I use the script it returns me "You have nothing to watch! (or AniList status isn't "WATCHING")"

Currently I have it set so to only show the anime with "Watching" status on your AniList when you try to watch an anime. Make sure that you've mapped a folder to an AniList ID for the anime you're trying to watch, and that on AniList, your status for that anime is "Watching". By the way, once you've mapped a folder, you can manually check `map.json` if you ever need to make changes to anything.

**Comment by hotsno** [comment]
About the site, I really don't deserve the compliment since it's basically a fork of [Bento](https://github.com/migueravila/Bento) but here is the repo for my site: https://github.com/hotsno/hotsno.me

**Comment by Estebiu** [comment]
Yep, I grasped what "anime folder means", but I have 4 different hard disks mounted in my home directory, and I didn't want this tool to scan everyfolder in my home directory by putting that as the anime path. Well, it's not a big issue.
![image](https://user-images.githubusercontent.com/61051731/179347366-a318c1ad-9e56-47d1-8a16-e1c2f6687ffe.png)
As you can see in this screenshot, I've mapped 3 anime folder in my map.json, and in my anilist profile 2 of them are set as "watching" (FLCL and How Heavy are Your..ecc) but the script doesn't let me watch them, at this moment. And yeah, I've added the script to mpv.

P.S Thanks for the repo of your website :D

**Comment by hotsno** [comment]
Did you add the token to config.json? It should’ve prompted you to do that the first time the script ran.

**Comment by hotsno** [comment]
I’ll add support for multiple anime folders when I get the time

**Comment by Estebiu** [comment]
> Did you add the token to config.json? It should’ve prompted you to do that the first time the script ran.
Yes, I did add it.

**Comment by hotsno** [comment]
Closing the issue as the issue @Estebiu was having was solved by redownloading the project. I should probably add some logging to this project to make debugging easier 😅

**Comment by hotsno** [comment]
Never actually added this feature, don't know why I closed this
